docs/guides/envvars.md

```diff
@@ -54,6 +54,27 @@ Note that `dotenv` is only for development, you should not use it in production,
 
 <docs-error>Do not commit your <code>.env</code> file to git, the point is that it contains secrets!</docs-error>
 
+If you're using the `@remix-run/cloudflare-pages` adapter, env variables work a little differently. When running locally, you'll need to manually pass your `.env` file to `wrangler` by doing this:
```

Comment:
I wonder if this would be better in the readme for `@remix-run/cloudflare-pages` rather than in the general docs for envvars?

Reply:
A Readme per adapter does seem like a good idea but they just seem to be boilerplate at the moment, so I left them as is.